tests/python_with/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -8,36 +8,15 @@ with open('tmp/index.html'):
     print('great')
 
 with A() as X, B() as Y, C() as Z:
-    do_something()~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    do_something()
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 with open("tmp/index.html") as f:
     index = f.read()
 
 with open("tmp/index.html"):
     print("great")
 
-with A() as X:
```

Comment:
and as a result of the build changes, we now automatically flag unnecessary 2/3 inconsistencies like these

(this was the only one)